common/src/main/java/org/apache/uniffle/common/util/RssUtils.java

```diff
@@ -272,6 +273,29 @@ public static Roaring64NavigableMap cloneBitMap(Roaring64NavigableMap bitmap) {
     return clone;
   }
 
+  public static Roaring64NavigableMap toBitmap(Set<Long> sets) {
```

Comment:
Could we wrap the the bitmap as a set to avoid unnecessary convention?

Reply:
It will be invoked by only one side. and it's not a hot point. Maybe this is not necessary